### PR TITLE
Port to PHPOffice Spreadsheet

### PR DESCRIPTION
**Pull request by infomaniac50**
### Summary of Changes
- Convert all usages of PHPExcel to PHPSpreadsheet
- PortPHP/Excel should really just be for proper spreadsheet files. CSV is served much better by PortPHP/Csv 
- 100% code coverage in unit tests.

### CODE REVIEW

**Comment by infomaniac50** [comment]
There are no more commits for this pull request.

File: src/SpreadsheetReader.php [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+use Port\Reader\CountableReader;
+
+/**
+ * Reads Spreadsheet files with the help of PHPSpreadsheet
+ *
+ * PHPSpreadsheet must be installed.
+ *
+ * @author David de Boer <david@ddeboer.nl>
```
Comment (ddeboer) [inline src/SpreadsheetReader.php]: Feel free to write your own name here. 😄

File: src/SpreadsheetReader.php [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+use PhpOffice\PhpSpreadsheet\IOFactory;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+use Port\Reader\CountableReader;
+
+/**
+ * Reads Spreadsheet files with the help of PHPSpreadsheet
+ *
+ * PHPSpreadsheet must be installed.
```
Comment (ddeboer) [inline src/SpreadsheetReader.php]: This line can be removed (probably a left-over from before PortPHP): the package already depends on PhpSpreadsheet.

**Comment by ddeboer** [comment]
Looks good to me. While there’s some duplication with portphp/excel, I don’t think that’s really a problem as that package can be deprecated and phased out.

@Baachi @sagikazarmark Do you agree with merging?

**Comment by jverdeyen** [comment]
@ddeboer any feedback on this PR? Would love to see this merged 👍🏼!

**Comment by jverdeyen** [comment]
Any update? @ddeboer @Baachi @sagikazarmark

**Comment by infomaniac50** [comment]
@ddeboer @Baachi @sagikazarmark I would like to see this merged as well. Is there anything left for me to do?

**Comment by Baachi** [comment]
Merged thanks!

**Comment by Baachi** [comment]
I also tagged version 1.0.0-alpha

**Comment by jverdeyen** [comment]
Thanks!

**Pull request merged**